lisp/obsolete/ess-noweb-mode.el

```diff
@@ -1789,6 +1789,32 @@ thread."
       (ess-noweb-tangle-thread chunk-name buffer))))
                                         ;menu functions
 
+;;;###autoload
```

Comment:
Shall we remove this autoload and file associations? If we don't people might not noticie the difference.

Reply:
I think the autoloads don't get generated from the obsolete/ directory but will double-check.

Reply:
They don't get generated by our makefile.